src/cockpit/bridge.py

```diff
@@ -125,9 +127,9 @@ def get_os_release():
             os_release[k] = v_parsed
         return os_release
 
-    def do_init(self, message: Dict[str, object]) -> None:
-        superuser = message.get('superuser')
-        if isinstance(superuser, dict):
+    def do_init(self, message: JsonObject) -> None:
+        with contextlib.suppress(JsonError):
+            superuser = message.get_object('superuser')
```

Comment:
This would silently ignore ill-formed superuser objects in the init message (that comes from ws, right?). It feels to me that this should be somewhere between "logged very loudly" and "refuse to operate because something obviously went wrong". Or would this already throw an error if the 'superuser' key doesn't exist at all?